core/server.ts

```diff
@@ -10,11 +10,11 @@ app.use(bodyParser.json());
 const loader = new Loader();
 loader.initRoute(app);
 loader.runMigrate(db)
-    .then(res => {
+    .then(() => {
         /**
          * TODO Change on logger
          */
-        console.log('*** Migrate Done ****');
+        console.log('Migrate Done.');
```

Comment:
Strange name, maybe `migration` is better. Also `console.log` should be replaced with logger